public/js/app.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+function getRedditPage(threadName) {
+  function redditListener() {
+    contentArea.innerHTML = '';
+    let responseDataArr = JSON.parse(this.responseText).data.children;
+    let defaultImg = 'https://i.redd.it/rq36kl1xjxr01.png';
+
+    for (let i = 0; i < responseDataArr.length; i++) {
+      let dataTitle = responseDataArr[i].data.title;
+      if (dataTitle.length === 0) {
+        dataTitle = '[Untitled]';
+      }
+      let dataAuthor = 'u/' + responseDataArr[i].data.author;
+      let dataCreated = responseDataArr[i].data.created;
+      let dataLikes = responseDataArr[i].data.ups.toLocaleString('en-US');
+      let dataText = responseDataArr[i].data.selftext;
+      let dataImage = responseDataArr[i].data.url;
+
+      // calcluate time since posting
+      let timeStamp = Math.floor(new Date().getTime() / 1000.0);
```

Comment:
You could have used moment.js here to do this work for you. Or did you want to figure out how to do it on your own?